recipes/astro-informatics-so3/all/conanfile.py

```diff
@@ -57,12 +59,18 @@ def generate(self):
         tc.variables["conan_deps"] = False
         tc.variables["CONAN_EXPORTED"] = True
         tc.variables["BUILD_TESTING"] = False
+        tc.variables["CMAKE_WINDOWS_EXPORT_ALL_SYMBOLS"] = True
         tc.generate()
         deps = CMakeDeps(self)
         deps.set_property("fftw", "cmake_target_name", "FFTW3::FFTW3")
         deps.generate()
 
+    def _patch_sources(self):
```

Comment:
I still prefer asking to the upstream supporting shared, instead of forcing it here. Changing it will generate a content that's not expect by upstream's users.

Reply:
I don't prefer the asking part, unless I'm personally invested in the project, tbh. :grin:
I can revert the forced shared libs support.

Reply:
Reverted to static-only.